### DISCUSSION TITLE
[vote ended on 2023-08-30] Update Python requirements for collections included in Ansible

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#268] [Votes done]
Vote +1 to merge https://github.com/ansible/ansible-documentation/pull/281, which updates the requirements on Python support for collections included in Ansible, or vote -1 to keep the current version (so we can discuss another update).

Discussion: https://github.com/ansible-community/community-topics/issues/251

**Comment by felixfontein** [comment]
+1

**Comment by sean-m-sullivan** [comment]
I think the only feedback is that the github matrix of the correct combinations should be maintained somewhere, similar to 
https://github.com/ansible-network/github_actions/blob/main/.github/workflows/sanity.yml

That way there is a known standard/combination to go off of, as even i forget sometimes and have to go hunt down the tables

**Reply by felixfontein** [reply]
This is for voting only, not for discussion. Please use the associated issue for discussions about this vote (https://github.com/ansible-community/community-topics/issues/251). (Also see https://github.com/ansible-network/github_actions/issues/120 on that specific workflow and why it should **not** be done that way for sanity and unit tests. The matrix for unit and sanity tests should only contain the ansible-core versions itself.)

**Comment by gotmax23** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by aretrosen** [comment]
+1

**Comment by spyrexd** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by sean-m-sullivan** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by cidrblock** [comment]
+1, ty for improving it

**Comment by jamescassell** [comment]
+1
